### PR TITLE
[WFLY-17581] Prolong timeout because of intermittent test failures

### PR DESCRIPTION
**Pull request by mchoma**
https://issues.redhat.com/browse/WFLY-17581

### CODE REVIEW

**Comment by mchoma** [comment]
@jamezp could you please have a look? I believe you was tweaking RestEasy testsuite with something similar.

File: testsuite/integration/microprofile-tck/rest-client/pom.xml [hunk 1]
```diff
@@ -476,7 +476,7 @@
                         <jboss.install.dir>${basedir}/target/wildfly</jboss.install.dir>
                         <jboss.server.config.file.name>standalone-microprofile.xml</jboss.server.config.file.name>
                         <!-- -Dts.include.servlet.4=true is needed due to https://github.com/eclipse/microprofile-rest-client/issues/346 -->
-                        <microprofile.jvm.args>${microprofile.jvm.args} -Dts.include.servlet.4=true -Dorg.eclipse.microprofile.rest.client.tck.timeoutCushion=50000 -Dorg.wildfly.unsupported.skip.jakarta.transformer=true</microprofile.jvm.args>
+                        <microprofile.jvm.args>${microprofile.jvm.args} -Dts.include.servlet.4=true -Dorg.eclipse.microprofile.rest.client.tck.timeoutCushion=90000 -Dorg.wildfly.unsupported.skip.jakarta.transformer=true</microprofile.jvm.args>
```
Comment (mchoma) [inline testsuite/integration/microprofile-tck/rest-client/pom.xml:479]: I can introduce some property here instead of hard-coded value.

**Review by jamezp — APPROVED** [review]
This seems fine. I do think we should also try upgrading the httpclient as well.

**Review by jbliznak — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @mchoma and @jbliznak

**Pull request merged**